### ISSUE TITLE
Persona: Alt of the image under Persona examples  not descriptive.

### ISSUE THREAD

**Issue by aneeshack4** [OfficeDev/office-ui-fabric-react#10853]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Alt of image under 'Rendering custom coin' section is not descriptive as reported by bug 3 of #7643 - looked at other Personas too on that page and some of them have empty strings for `alt`.

### Please provide a reproduction of the issue in a codepen:
![image](https://user-images.githubusercontent.com/4161791/66877611-18a99800-ef84-11e9-9d88-7594b8d98bc6.png)

#### Actual behavior:

Alt of image under 'Rendering custom coin' section is not descriptive.

#### Expected behavior:

Proper and descriptive alt should be defined which should be related to visible persona like: Ted Randall, status is available.

**Comment by micahgodbolt** [comment]
the controls are fine, we just need to improve our docs examples using best practices
